src/Categories/Morphism/Reasoning.agda

```diff
@@ -6,5 +6,18 @@ open import Categories.Category
 
 module Categories.Morphism.Reasoning {o ℓ e} (C : Category o ℓ e) where
 
+-- some items are defined in sub-modules
 open import Categories.Morphism.Reasoning.Core C public
 open import Categories.Morphism.Reasoning.Iso C public
+
+open Category C
+open Definitions C
+open HomReasoning
+
+-- create a commutative square from an equivalence
```

Comment:
Again, I'm wondering why this is in the PR.

Reply:
I think because `toSquare` was buried in some Quiver-related file, and it made most sense to move it here at the same time.